doc/docs/master/deploy-manage/manage/s3gateway/deploy-s3gateway-sidecar.md

```diff
@@ -0,0 +1,92 @@
+# Create an S3-enabled Pipeline
+
+Pachyderm can deploy the S3 gateway either together with the `pachd` pod or
+as a sidecar next to your pipeline worker pod. The former is
+typically used when you need to configure an ingress or egress with
+object storage tooling, such as MinIO, boto3, and others. The latter
+is needed when you use Pachyderm with external data processing
+platforms that interact with object stores but do not work with local
+file systems, such as Kubeflow or Apache Spark.
```

Comment:
we're using pod and sidecar, both of which are pretty k8s-specific terminology. I wonder - how much k8s knowledge should we expect from users? Is it reasonable to assume that using that terminology will help them grok the s3g sidecar?

Reply:
@ysimonson how are you suggesting to rephrase this?

Reply:
It depends on how much k8s knowledge we should expect from the users

Reply:
Added some high-level intro. Still, I think we should keep the details about the K8s internals.